docs/HowToFAQ/ERROR_NoLHS.md

```diff
@@ -9,7 +9,10 @@ Here is example code that produces this error:
 
 
 When a _such-that_ (`:|`) initialization is used, Dafny must be able to establish that there is at least one value
-that satisfies the predicate given on the RHS. In this case if the map `m` is empty, there is no such value,
+that satisfies the predicate given on the RHS. 
+That is, in this case, it must prove
+`assert exists k | k in m;`.
```

Comment:
```suggestion
`assert exists k :: k in m;`.
```
Sorry, my bad, I just checked, and the single `|` syntax is invalid. It should but it's not.